### PR TITLE
feat: add products page

### PR DESCRIPTION
**Pull request by anik31**
Products page for Ecommerce project.
product.html contains the HTML code & style.css contains the CSS code.

Deploy preview - https://deploy-preview-4--vacay-store.netlify.app/product

### CODE REVIEW

**Review by ApurvaSawant11 — APPROVED** [review]
Looks good. Just one change. Radio buttons are not working.

File: product.html [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+                <li class="input input-checkbox-radio">
+                    <input type="checkbox">
+                    <label for="">Suitcase</label>
+                </li>
+            </ul>
+            <ul>
+                <h4>Rating</h4>
+                <li class="input input-checkbox-radio">
+                    <input type="radio">
+                    <label for="">4 Stars & above</label>
```
Comment (ApurvaSawant11) [inline product.html:64]: All the radio buttons are getting clicked. Add the same value for the `name` attribute to all radio inputs.

**Review by ArjunJunna — APPROVED** [review]
Need changes in the radio button. Rest everything looks good

File: product.html (hunk 1 above)
Comment (anik31) [reply product.html:64]: @ApurvaSawant11 can you check now, I have done the changes

Comment (ApurvaSawant11) [reply product.html:64]: Yes it's working now

**Comment by anik31** [comment]
> Need changes in the radio button. Rest everything looks good

I have fixed it @ArjunJunna

**Comment by anik31** [comment]
Thanks for reviewing @ApurvaSawant11 @ArjunJunna

**Pull request merged**